### ISSUE TITLE
Suggestion: use actual steam fluid

### ISSUE THREAD

**Issue by DaeTheDerg** [RealAntEngineer/CROWNS#2]
Hiya! I've been hoping to have a mod that integrates steam production (railcraft, extreme reactors, nuclearcraft, etc.) with create mechanical power, I've always been frustrated that the regular steam engines don't actually produce or use any steam fluid, and therefore can't use steam from other sources, additionally, the ability to pipe steam all over to different engines with one big boiler would be brilliant!
the turbine from this mod looked like it was perfect, but unfortunately, it doesn't seem to accept steam either, just water after having gone through the heaters, which is a real shame
if the heaters could convert water into steam, and the turbine outlets ran on that instead of water, I feel the whole system would be a lot more adaptable to different needs!

**Comment by RealAntEngineer** [comment]
yeah, but that would be way harder to implement. but I might make a block to convert heated water into steam

**Comment by DaeTheDerg** [comment]
That sounds like a good medium!
If possible a block that converts the other way too would be great as well 💜

**Comment by bjkjhguktdftyuhguy** [comment]
> yeah, but that would be way harder to implement. but I might make a block to convert heated water into steam
honestly if it can be a steam separator/stage separation heat exchanger im all for it
